py_feels/binary/binary_classifier.py

```diff
@@ -12,25 +12,31 @@
 
 # we only need pandas, mongo and joblib to use the model
 import pandas as pd
+import datetime
+from sqlalchemy import create_engine
 from sklearn.externals.joblib import load
 
 # load fitted model
 model = load('./model/binary_classifier.pckl')
 
-# read amazon training data for testing loaded model
-amazon = pd.read_csv(
-        './training/UCI_training_data/amazon_cells_labelled.txt', 
-        sep="\t", 
-        names=['sentence', 'sentiment'], 
-        dtype={'sentence':str, 'sentiment':int}
-)
+# connect to  SQL database, query for last item in 'tweets' table
+engine = create_engine('postgresql+psycopg2://wojak:@localhost/bit_feels_dev')
+query  = "SELECT * FROM tweets WHERE id=(SELECT max(id) FROM tweets)"
+tweets = pd.read_sql(query, engine)
 
-# predict sentiments in the amazon data
-predictions = model.predict(amazon['sentence'])
-
-# print accuracy, 1 is perfect
-accuracy = (predictions == 2*amazon['sentiment']-1).sum()/predictions.size
-print("Accuracy of %.3f on amazon data" % accuracy)
+# predict sentiments in dataframe
+sentiments = model.predict(tweets.text.values)
 
+# construct feels DataFrame for passing back to SQL
+feels = pd.DataFrame()
+feels['sentiment']   = sentiments
+feels['classifier']  = dict(model.steps)['clf'].__class__.__name__
+feels['tweet_id']    = tweets.tweet_id.values
 
+# store prediction time
+time_now = str(datetime.datetime.now())
+feels['inserted_at'] = time_now
+feels['updated_at']  = time_now
```

Comment:
Can this be `nil`?

Reply:
it comes from the system time, so I don't think so